### ISSUE TITLE
alt attribute is missing space in front

### ISSUE THREAD

**Issue by podcasthosting** [dietercoopman/smart#21]
Hi Dieter,

first off thank you for this very interesting and promising package! I have just recently discovered it through Laravel News.

For web accessibility and SEO I always try to add an _alt_ tag to my images. You support (common) HTML attributes as part of the templates. Unfortunately the rendered output lacks a space in front of the _alt_ tag making the rendered HTML invalid.

`<x-smart-image alt="My image" width="255" height="160" src="/images/myimage.jpg" data-src="my-image.jpg" />`

Expected:

`<img src='/smart/my-image.jpg' alt='My image' width='255' height='160' >`

Result:

`<img src='/smart/my-image.jpg'alt='My image' width='255' height='160' >`

Would be nice if you could fix this. Thank you!

**Comment by podcasthosting** [comment]
I just discovered this is not exclusive to the alt tag but to all attributes added additionally.

**Comment by dietercoopman** [comment]
Hi @podcasthosting , thx for this bug report.  I've fixed it in version 1.5.1 

Have a nice day !
